docs/api/modules/App-Configuration.md

```diff
@@ -286,6 +286,14 @@ if (!global.BROWSER) {
 }
 ```
 
+### Caching
+
+When the service worker is enabled, the Caches API is working on every `fetch` event
```

Comment:
So we can safely say that this caching functionality is completely "revertible" and opt-in along just like the core PWA feature is today?

Reply:
Yes, when removing the service worker, the cache is completely cleared, restarting the service worker in recovery mode and back, it will trigger a reset of the cache. It isn't an opt-in is turned on automatically.

Reply:
Gotcha 👍